posts/2024-05-10_new_admiral_ex.../new_admiral_extension_packages_admiralpeds_coming_soon.qmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "New {admiral} extension packages: {admiralpeds} & more coming soon!"
+author:
+  - name: Fanny Gautier
+  - name: Lina Patil
+description: "{admiralpeds} & more announcements"
```

Comment:
Can we have a slightly more verbose description please? Maybe a couple of sentences? Just thinking about the newsletter we send out - a slightly longer blurb would be helpful, thanks :)

Reply:
Could you please double check the updates ? thanks.